### PR TITLE
Resolve issue of error when connecting to mssql with docker image or exe

### PR DESCRIPTION
**Pull request by EmmaZhu**
Thanks for contribution! Please go through following checklist before sending PR.

### PR Branch Destination

- For Azurite V3, please send PR to `main` branch.
- For legacy Azurite V2, please send PR to `legacy-dev` branch.

### Always Add Test Cases

Make sure test cases are added to cover the code change.

### Add Change Log

Add change log for the code change in `Upcoming Release` section in `ChangeLog.md`.

### Development Guideline

Please go to CONTRIBUTION.md for steps about setting up development environment and recommended Visual Studio Code extensions.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -33,8 +31,11 @@ COPY package*.json LICENSE NOTICE.txt ./
 
 COPY --from=builder /opt/azurite/dist/ dist/
 
-RUN npm config set unsafe-perm=true && \
-  npm install -g --loglevel verbose
+RUN npm pkg set scripts.prepare="echo no-prepare"
+
+RUN npm ci --unsafe-perm
```
Comment (XiaoningLiu) [inline Dockerfile:36]: Do we need to additionally add `npm ci`?

Comment (EmmaZhu) [reply Dockerfile:36]: After Node16, the `install -g` only copies content under local folder to global folder, it didn't install anything. `npm ci` is to install dependencies into local folder first

**Review by blueww — APPROVED** [review]
Thanks for the fix!

1. Would you please share the test result of the fix with docker/exe on SQL and loki?
2. It's better to cover SQL server in test pipeline. (currently on my sql covered). If you don't want to add with this PR, you can open an issue to track it.

**Comment by blueww** [comment]
@EmmaZhu 

Would you please help to look at PR : https://github.com/Azure/Azurite/pull/2380/files？
It looks also try to upgrade node version in docker. However, the build script change in different way from this PR.

**Review by XiaoningLiu — APPROVED** [review]

**Pull request merged**